templates/NOTES.txt

```diff
@@ -1,28 +1,19 @@
 1. In Helm Version <2.7 following job needs to be cleaned manually.
```

Comment:
Also, this still seems to be needed with helm 2.7.

Reply:
We need to verify this as Helm 2.7 includes support of `"helm.sh/hook-delete-policy"` and should clean up automatically.